tests/compiler/test_basics.py

```diff
@@ -474,9 +474,10 @@ def get_one() -> "int":
     )
     with warnings.catch_warnings(record=True) as w:
         compiler.do_compile()
-        assert len(w) == 1 if decorator else len(w) == 0
-        if len(w):
-            warning = w[0]
+        # first warning is about the use of V1 modules that are deprecated.
+        assert len(w) == 2 if decorator else len(w) == 1
+        if len(w) == 2:
+            warning = w[1]
```

Comment:
These assertions are too strict. If at some point we introduce an additional warning somewhere, this test case will fail because `len(w) == 3` instead of `len(w) == 2`.